conf/vsc_ugent.config

```diff
@@ -42,7 +33,25 @@ process {
     stageInMode = "symlink"
     stageOutMode = "rsync"
     errorStrategy = { sleep(Math.pow(2, task.attempt ?: 1) * 200 as long); return 'retry' }
-    maxRetries    = 5
```

Comment:
Please don't remove this as it's a failsafe for inconsistencies from the scheduler. Maybe lower this to 3 as a compromise? You can always turn it off in your own configs if you don't want that